src/material/badge/badge.ts

```diff
@@ -129,9 +130,15 @@ export class MatBadge extends _MatBadgeBase implements OnInit, OnDestroy, CanDis
   /** Visible badge element. */
   private _badgeElement: HTMLElement | undefined;
 
+  /** Inline badge description. Used when the badge is applied to non-interactive host elements. */
+  private _inlineBadgeDescription: HTMLElement | undefined;
```

Comment:
The badge itself is already an inline element in the host element, but it has an `aria-hidden`. Couldn't we get the same result by removing the `aria-hidden` in non-interactive hosts?